tutorial/conan_repositories/setting_up_conan_remotes/artifactory/artifactory_ce_cpp.rst

```diff
@@ -10,24 +10,24 @@ API, a generic CLI tool and generic repositories to host any kind of source or b
 artifact.
 
 This is a very brief introduction to Artifactory CE. For the complete Artifactory CE
-documentation, visit `Artifactory docs <https://www.jfrog.com/confluence/>`_.
+documentation, visit `Artifactory docs <https://jfrog.com/help/>`_.
 
 Running Artifactory CE
 ----------------------
 
 There are several ways to run Artifactory CE:
 
-* **Running from a docker image**. Just run:
+* **Running from a Docker image:**
 
 .. code-block:: bash
     
-    $ docker run --name artifactory -d -p 8081:8081 -p 8082:8082 docker.bintray.io/jfrog/artifactory-cpp-ce:latest
+    $ docker run --name artifactory -d -p 8081:8081 -p 8082:8082 releases-docker.jfrog.io/jfrog/artifactory-cpp-ce:latest
```

Comment:
In fact, this matches with what https://jfrog.com/help/r/jfrog-installation-setup-documentation/install-artifactory-single-node-with-docker indicates